Source/Fuse.Elements/Resources/Exif/Exif.uno

```diff
@@ -0,0 +1,190 @@
+using Uno;
+using Uno.IO;
+using Uno.Compiler.ExportTargetInterop;
+
+namespace Fuse.Resources.Exif
+{
+	[Flags]
+	public enum ImageOrientation
+	{
+		Identity = 0,
+		Rotate90 = 1,
+		Rotate180 = 2,
+		Rotate270 = 3,
+		FlipVertical = 1 << 2,
+	}
+
+	public struct ExifData
+	{
+		public readonly ImageOrientation Orientation;
+
+		public static ExifData FromByteArray(byte[] buffer)
+		{
+			if defined(Android)
+				return ExifAndroidImpl.FromByteArray(buffer);
+			if defined(iOS)
+				return ExifIOSImpl.FromByteArray(buffer);
+			if defined(DotNet)
+				return ExifDotNetImpl.FromByteArray(buffer);
+			else
+				throw new Exception("Exif is not supported by this platform");
+		}
+
+		internal ExifData(int orientation)
+		{
+			//This is based on the vertical flip being applied before the rotation
```

Comment:
I think this would be better written as proper documentation on the ImageOrientation enum instead, no?